torch/nested/_internal/ops.py

```diff
@@ -36,14 +36,18 @@ def _wrap_jagged_dim(
     return _outer_to_inner_dim(ndim, wrapped) if convert_to_inner_dim else wrapped
 
 
-def _wrap_jagged_dims(ndim, dims, op_name, ragged_idx=1):
+def _wrap_jagged_dims(ndim, dims, op_name, ragged_idx=1, allow_batch_dim=False):
```

Comment:
nit: since you aren't using allow_batch_dim, you could probably remove the changes to this file

Reply:
thanks! removed the condition for `allow_batch_dim`